### ISSUE TITLE
Add option to stop moving of page by touching

### ISSUE THREAD

**Issue by Gamer750** [edeso/bubble2#4]
Add option to stop moving of page by touching its pretty annoying

**Comment by edeso** [comment]
dunno what you mean. pls provide a video or illustration.

**Comment by edeso** [comment]
no. these gesture are essentially for paging. why do you touch the screen if you dont want to switch the page?

**Comment by Gamer750** [comment]
> no. these "gestures" are essentially for paging. you can either swipe or tip on the left/right side
> 
> why do you touch the screen if you dont want to switch the page?

It would be good if it had toggle button to stop moving page by tapping edge

**Comment by edeso** [comment]
> > no. these "gestures" are essentially for paging. you can either swipe or tip on the left/right side
> > why do you touch the screen if you dont want to switch the page?
> 
> It would be good if it had toggle button to stop moving page by tapping edge

again. why do you tip the screen if you do not want to do anything? and yeah, option settings would be nice but are not high on my list. sorry

**Comment by edeso** [comment]
sorry to hear. unfortunately adding settings is currently not high on my list of things to do.
